packages/truffle-db/src/workspace/test/index.ts

```diff
@@ -17,27 +18,53 @@ const fixturesDirectory = path.join(
   "fixtures"
 );
 
+class WorkspaceClient {
+  private workspace: Workspace;
+
+  constructor () {
+    this.workspace = new Workspace();
+  }
+
+  async execute (request, variables = {}) {
+    const result = await graphql.execute(
+      schema,
+      request,
+      null, // root object, managed by workspace
+      { workspace: this.workspace }, // context vars
+      variables
+    );
+
+    return result.data;
```

Comment:
I'm doing this for convenience but it might be a bad idea because we're going to want to test for errors eventually. I figure we can always change it later, since it won't be _too bad_ to add all the extra destructuring back in, but feel free to disagree :)

Reply:
I'm ok with leaving it like this for now!